### DISCUSSION TITLE
WSLu now available for Clear Linux

### DISCUSSION THREAD

**Discussion by WSLUser** [microsoft/WSL#8702] [Show and tell]
The great author of wslu, @patrick330602 has made Clear Linux another distro to use! See https://blog.wslutiliti.es/2022/08/08/wslu-now-available-on-clearlinux/

I have requested it to be officially added to the Clear Linux repository on their distribution page: https://github.com/clearlinux/distribution so be sure to upvote if you enjoy using Clear Linux.

**Comment by WSLUser** [comment]
@nunix I bet you're excited about this news once you see this.

**Reply by nunix** [reply]
I see you my friend, and you're totally making my day 😄
